sql/core/src/main/scala/org/apache/spark/sql/execution/Columnar.scala

```diff
@@ -260,9 +260,9 @@ private object RowToColumnConverter {
       case BooleanType => BooleanConverter
       case ByteType => ByteConverter
       case ShortType => ShortConverter
-      case IntegerType | DateType => IntConverter
+      case IntegerType | DateType | YearMonthIntervalType(_, _) => IntConverter
```

Comment:
Please, use _: YearMonthIntervalType. In this ways, your code won't depend on the number of parameters. Also, see https://github.com/databricks/scala-style-guide#pattern-matching :
_"If the only goal is to match on the type of the object, do NOT expand fully all the arguments, as it makes refactoring more difficult and the code more error prone."_

Reply:
Thank you very much for the reminder. Done.